src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -52,11 +52,11 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
 
         CommandResult commandResult;
 
-        Command command = addressBookParser.parseCommand(commandText);
+        Command command = patientListParser.parseCommand(commandText);
         commandResult = command.execute(model);
 
         try {
-            storage.saveAddressBook(model.getAddressBook());
+            storage.savePatientList(model.getPatientList());
             storage.saveAppointmentList(model.getAppointmentList());
```

Comment:
Do you think we need to rename this also?

Reply:
Why do u think so?